controllers/fenceagentsremediation_controller.go

```diff
@@ -287,6 +287,16 @@ func (r *FenceAgentsRemediationReconciler) updateStatus(ctx context.Context, far
 	return nil
 }
 
+func getNodeName(far *v1alpha1.FenceAgentsRemediation) string {
+	if far.GetAnnotations() == nil {
+		return far.GetName()
+	} else if nodeName, isNodeNameAnnotationExist := far.GetAnnotations()[commonAnnotations.NodeNameAnnotation]; isNodeNameAnnotationExist {
```

Comment:
NIT: no need for `else` block after we return a value. Not sure which one way is more readable, but the else is redundant